packages/reports/addon/components/dimension-selector.js

```diff
@@ -22,6 +23,37 @@ import layout from '../templates/components/dimension-selector';
 import { featureFlag } from 'navi-core/helpers/feature-flag';
 import { getDefaultTimeGrain } from 'navi-reports/utils/request-table';
 
+export const THROTTLE_TIME = 750; // milliseconds
+
+const ANIMATIONS = {
+  animation: 'animationend',
+  OAnimation: 'oAnimationEnd',
+  MozAnimation: 'animationend',
+  WebkitAnimation: 'webkitAnimationEnd'
+};
+
+/**
+ * Blur button at the end of the target's animation
+ * @param {Element} target - grouped-list__item-container--selected element for the clicked item that will play the animation
+ * @param {Element} button - grouped-list__item-label element that is the button that fired off the action
+ */
+export function BlurOnAnimationEnd(target, button) {
+  const listItemContainer = target.closest('.grouped-list__item-container--selected');
+  if (listItemContainer) {
+    // Detect the end of the css animation depending on browser and blur the button
+    let animationEndEvent;
+    for (let animation in ANIMATIONS) {
+      if (listItemContainer.style[animation] != undefined) {
+        animationEndEvent = ANIMATIONS[animation];
+      }
+    }
```

Comment:
will something like `listItemContainer.classList.add('some-animation-class-like-navi-column-config-item--last-added')` work?
then we might be able to get rid of the prefixes and the focus that creates this dotted border on firefox
<img width="135" alt="Screen Shot 2020-05-18 at 11 50 54 AM" src="https://user-images.githubusercontent.com/13946669/82248980-e6875c80-98fd-11ea-959f-a99b93dde11b.png">
if we need to restart the animation for the same item we can maybe do https://css-tricks.com/restart-css-animation/#article-header-id-0

Reply:
This was my first approach, but because these elements exist inside vertical collection, relying on the addition and removal of these elements to play the animations was causing animation replays on scroll. I can try to rework it by attaching a separate animation class, but I feel like it might just be easier to do this to fix the outline:
https://stackoverflow.com/questions/71074/how-to-remove-firefoxs-dotted-outline-on-buttons-as-well-as-links